serpent-moltres/full-core/pathmagic.py

```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.insert(1, "/home/roberto/github/mhtgr350-benchmark")
```

Comment:
Do you really need an entirely separate file for this? You already import sys in ``postprocessing.py``, so you'd really just be adding one line to that file.

Reply:
I don't need it, but PEP8 complains. If I did that my script would read:
```
import sys
sys.path.insert(1, "/home/roberto/github/mhtgr350-benchmark")
from auxiliary import plot_serpent_axial_collapse
```
If I do that, PEP8 throws an error: ```E402 module level import not at top of file```

Reply:
Okay, this will be resolved when you address issue #56